stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/common/internal/SnapToTopDataObserver.kt

```diff
@@ -51,6 +51,8 @@ internal class SnapToTopDataObserver(
             return
         }
 
```

Comment:
This is also relevant to the fix as previously it tried to scroll to the top even if there were no previous items and `layoutManager.findFirstVisibleItemPosition()` returned `-1`.